__tests__/tests.js

```diff
@@ -11,9 +11,9 @@ test('if invalid api base path, then 404', async () => {
   expect(response.status).toEqual(404);
 });
 
-test('if invalid path, then 501', async () => {
+test('if invalid path, then 404', async () => {
```

Comment:
this test was asserting the wrong status before.
I'll probably mark this release as a breaking change because of this